### PR TITLE
new test case with keyword FBHPDEF

### PR DESCRIPTION
**Pull request by tskille**
The keyword FBHPDEF is not supported in OPM Flow

### CODE REVIEW

File: model2/0C_BASE_FBHPDEF.DATA [hunk 1]
```diff
@@ -0,0 +1,839 @@
...
+
+-- This is test case for keyword FBHPDE. This model contains 5 wells. 3 producers controlled
+-- with keyword WCONHIST and two injectors controlled with keyword WCONINJH.  
+
+-- One of the injectors (INJ1) and two of the producers (PROD2 and PROD3) have reduced PI
+-- (see keyword WPIMULT). Bottomhole pressure for these wells are outside pressure 
+-- coverage in PVT table causing extrapolation.
+
+-- The keyword FBHPDEF is used to ensure that wbhp for all wells will stay above 50 bars and 
+-- below 500 bars during the simuation.
```
Comment (akva2) [inline model2/0C_BASE_FBHPDEF.DATA:16]: simulation

Comment (tskille) [reply model2/0C_BASE_FBHPDEF.DATA]: thanks, fixed

**Comment by tskille** [comment]
Small update of test case. Adding a short period with gas injection for prododucer PROD2.

**Review by bska — APPROVED** [review]
Thanks a lot.  This looks good to me and I'll merge into master.

**Pull request merged**